web/src/main/java/org/springframework/security/web/access/expression/WebSecurityExpressionRoot.java

```diff
@@ -39,6 +40,17 @@ public WebSecurityExpressionRoot(Authentication a, FilterInvocation fi) {
 		this.request = fi.getRequest();
```

Comment:
This constructor can use the newly added constructor `this(authentication, fi.getRequest())`

Reply:
Done.